src/main/resources/static/js/scripts.js

```diff
@@ -15,4 +15,19 @@ function disablePushConfirmation(){
         .attr("disabled", "disabled")
         .removeClass("hovering-purple-button pull-right buffer-top")
         .addClass("button_disabled pull-right buffer-top")
+}
+
+function sortByDateDesc(){
+    var dateArray = [];
+    $('#summaryTableBody tr').has('td').each(function() {
+        var arrayItem = {};
+        $('td', $(this)).each(function (index, item) {
+            arrayItem[index] = $(item).html();
+        })
+    });
+
+        alert(dateArray.toString);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this usage of alert(...). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=javascript%3AS1442)